utils/api/submissions.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { useUnfinishedSubmissions } from './submissions';
+import * as SWR from 'swr';
+
+jest.mock('swr');
+
+describe('residents APIs', () => {
+  describe('useResident', () => {
+    it('should work properly', () => {
```

Comment:
Looks like we're also doing some filtering on the response we get back. Could we add some unit tests around this?